custom/icds_reports/views.py

```diff
@@ -335,6 +337,34 @@ def get(self, request, *args, **kwargs):
         return JsonResponse(data=data)
 
 
+@method_decorator([login_and_domain_required], name='dispatch')
+class ServiceDeliveryDashboardView(BaseReportView):
+
+    def get(self, request, *args, **kwargs):
+        step, now, month, year, include_test, domain, current_month, prev_month, location, selected_month = \
+            self.get_settings(request, *args, **kwargs)
+
+        location_filters = get_location_filter(location, domain)
+        location_filters['aggregation_level'] = location_filters.get('aggregation_level', 1)
+        age_sdd = request.GET.get('ageSDD')
+
+        start, length, order_by_number_column, order_by_name_column, order_dir = \
+            get_datatables_ordering_info(request)
+        reversed_order = True if order_dir == 'desc' else False
+
+        data = get_service_delivery_dashboard(
```

Comment:
might want to change this to `get_service_delivery_data`. I think its a little clearer since this is just the data accessor

Reply:
Sure